src/Sylius/Behat/Resources/config/suites/ui.yml

```
@@ -26,6 +26,8 @@ imports:
     - ui/currency/currencies.yml
     - ui/currency/managing_currencies.yml
     - ui/currency/managing_exchange_rates.yml
+    - ui/errors/admin_error_page.yaml
+    - ui/errors/shop_error_page.yaml
```

I wonder if we should not divide the suits into admin and shop, as we did with the scenarios
```suggestion
    - ui/errors/admin/error_page.yaml
    - ui/errors/shop/error_page.yaml
```
but of course, not in this PR 🖖🏻